Source/WebCore/PAL/pal/spi/mac/NSMenuSPI.h

```diff
@@ -67,10 +72,17 @@ enum {
 
 @class QLPreviewMenuItem;
 
-@interface NSMenuItem ()
+@interface NSMenuItem (Staging_139537720)
```

Comment:
This would imply the entire category is for staging related to rdar://139537720, but that's not the case.